src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -156,7 +156,6 @@ public partial class SendPacketsElement
     public partial class Socket : System.IDisposable
     {
         public Socket(System.Net.Sockets.AddressFamily addressFamily, System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType) { }
-        public Socket(System.Net.Sockets.SocketInformation socketInformation) { }
```

Comment:
@CIPop  It was:
                    UnsafeSocketsNativeMethods.OSSOCK.WSAPROTOCOL_INFO protocolInfo = 
                        (UnsafeSocketsNativeMethods.OSSOCK.WSAPROTOCOL_INFO)Marshal.PtrToStructure<UnsafeSocketsNativeMethods.OSSOCK.WSAPROTOCOL_INFO>((IntPtr)pinnedBuffer);